$OpenBSD: patch-test-framework_cabal,v 1.1 2017/11/08 18:52:43 kili Exp $

Adapt dependency changes (manually tweaked via that webinterface on
hackage.haskell.org but not contained in the distfile).

Index: test-framework.cabal
--- test-framework.cabal.orig
+++ test-framework.cabal
@@ -47,10 +47,10 @@ Library
                                 Test.Framework.Utilities
 
         Build-Depends:          ansi-terminal >= 0.4.0, ansi-wl-pprint >= 0.5.1,
-                                base >= 4.3 && < 5, random >= 1.0, containers >= 0.1,
+                                base >= 4.3 && < 5, random >= 1.0 && < 2, containers >= 0.1,
                                 regex-posix >= 0.72,
-                                old-locale >= 1.0,
-                                time >= 1.1.2 && < 1.6,
+                                old-locale == 1.0.*,
+                                time >= 1.1.2 && < 1.9,
                                 xml >= 1.3.5, hostname >= 1.0
 
         Extensions:             CPP
